pypodcastparser/Podcast.py

```diff
@@ -357,8 +357,13 @@ def set_published_date(self, tag):
         try:
             self.published_date = tag.string
             self.published_date_string = tag.string
-            pubdate = datetime.datetime.strptime(self.published_date, "%a, %d %b %Y %H:%M:%S %Z")
-            self.published_date = datetime.datetime.strftime(pubdate, "%Y-%d-%m, %H:%M:%S")
+            a = tag.string.split(":")
+            b = a[2]
```

Comment:
Can we employ more accurate variable-naming, and maybe a comment explaining why this is necessary?

Reply:
good point, will address